packages/env/src/constants/directus-variables.ts

```diff
@@ -123,6 +123,10 @@ export const DIRECTUS_VARIABLES = [
 	'STORAGE_.+_KEY_FILENAME',
 	'STORAGE_.+_BUCKET',
 	'STORAGE_.+_HEALTHCHECK_THRESHOLD',
+	'STORAGE_.+_CONNECTION_TIMEOUT',
+	'STORAGE_.+_SOCKET_TIMEOUT',
+	'STORAGE_.+_MAX_SOCKETS',
+	'STORAGE_.+_KEEP_ALIVE',
```

Comment:
```suggestion
```

Reply:
Can you tell why we should remove these ones? Since these configurations now exists (at least for S3 for now), I thought they needed to be declared here, otherwise why we have other variables like `STORAGE_.+_SERVER_SIDE_ENCRYPTION`? 🤔

Reply:
Hmm, there's definitely inconsistency. But I think driver specific envs should not be within directus variables. 🤔

`apiEndpoint` in the GCS driver, `serviceRole` in Supabase driver and many others are not in this list.